files/__addonLocation__/package.json

```diff
@@ -33,7 +33,6 @@
     <% if (typescript) { %>"@babel/preset-typescript": "^7.18.6"<% } else { %>"@babel/eslint-parser": "^7.19.1"<% } %>,
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-proposal-decorators": "^7.20.13",
-    "@babel/plugin-syntax-decorators": "^7.17.0",
```

Comment:
To my knowledge, `@babel/plugin-syntax-decorators` is unused in a v2 addon and is [not needed according to Babel](https://babeljs.io/docs/babel-plugin-syntax-decorators).

> Syntax only
>
> It's unlikely you want to use this plugin directly as it only enables Babel to parse this syntax. Instead, use [plugin-proposal-decorators](https://babeljs.io/docs/babel-plugin-proposal-decorators) to _both_ parse and transform this syntax.